CHANGELOG.md

```diff
@@ -4,6 +4,15 @@ All notable changes will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [5.0.0] - 2022-04-01
+### Changed
+- hyphens are no longer allowed in variable names in yaml files
```

Comment:
Could we highlight this as a backwards incompatible change? Maybe with a `**breaking**` in front?